api/src/main/java/org/openmrs/module/fhir2/api/FhirEncounterService.java

```diff
@@ -19,8 +17,6 @@
 
 public interface FhirEncounterService extends FhirService<Encounter> {
 	
-	Encounter get(@NotNull String uuid);
```

Comment:
Please actually leave this on the interface. I'm intending for the Javadocs to go on interfaces where possible.

Reply:
But this is unnecessary repetition. This method is already present in the BaseFhirService, why do we want to declare it here as well?

Reply:
To have a per-type Javadoc, which should be, but hasn't been the standard. In most cases it doesn't matter, but it some cases it might.